client/src/components/Form/Elements/FormSelectMany/FormSelectMany.vue

```diff
@@ -65,7 +65,7 @@ const searchRegex = computed(() => {
 /** Wraps value prop so it can be set, and always returns an array */
 const selected = computed({
     get() {
-        return Array.isArray(props.value) ? props.value : [props.value];
+        return props.value === null ? [] : Array.isArray(props.value) ? props.value : [props.value];
```

Comment:
this double ternary op is giving me creeps, maybe just me though :D

Reply:
Ok fixing that

Reply:
thanks @ahmedhamidawan !